app-crypt/stegsnow/stegsnow-20130616.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Hides information in text files using trailing whitespace"
+DESCRIPTION="SNOW (Steganographic Nature Of Whitespace) is a steganography \
+tool which conceals messages in ASCII text by appending spaces to the end of \
+lines. Has an option to encrypt the message first to further secure it."
+HOMEPAGE="http://www.darkside.com.au/snow/"
+COPYRIGHT="1999-2013 Matthew Kwan"
```

Comment:
The software already existed in 1998.
(Can you update COPYRIGHT to fix that? Thanks!)

BTW, CHECKSUM_SHA256 should come before SOURCE_DIR.

(See [HaikuPorter Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering))